src/Tickets/Commerce/Provider.php

```diff
@@ -28,13 +28,15 @@ class Provider extends tad_DI52_ServiceProvider {
 	 */
 	public function register() {
 
+		$this->container->make( Payments_Tab::class );
```

Comment:
We should turn that class into a provider to avoid having to call it.
```suggestion
		$this->container->register( Payments_Tab::class );
```